rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
```ruby
    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
```

- [ ] After making all the changes in the both book and person class you would need to update the highlighted code as you can see above :+1: